docs/mp/reactivemessaging/kafka.adoc

```diff
@@ -83,6 +93,69 @@ public PublisherBuilder<String> produceToKafka() {
 }
 ----
 
+== NACK Strategy
+
+|===
+| Strategy | Description
+| Kill channel | Nacked message sends error signal and causes channel failure so Messaging Health check can report it as DOWN
+| DLQ | Nacked messages are sent to specified dead-letter-queue
+| Log only | Nacked message is logged and channel continues normally
+|===
+
+=== Kill channel
+Default NACK strategy for Kafka connector. When
+
+=== Dead Letter Queue
+
+Sends nacked messages to error topic, link:https://en.wikipedia.org/wiki/Dead_letter_queue[DLQ] is well known pattern for dealing with unprocessed messages.
+
+Helidon can derive connection settings for DLQ topic automatically if the error
+topic is present on the same Kafka cluster.
+Serializers are derived from deserializers used for consumption
+`org.apache.kafka.common.serialization.StringDeserializer` >
+`org.apache.kafka.common.serialization.StringSerializer`.
+In this ideal case only the name of error topic is needed.
+
+[source,yaml]
+.Example of derived DLQ config:
+----
+mp.messaging:
+  incoming:
+    my-channel:
+      nack-dlq: dql_topic_name
+----
+
+In case custom connection is needed, all the usual producer configuration is possible
```

Comment:
If a custom connection is needed, then use the 'nack-dlq' key for all of the producer configuration.